### DISCUSSION TITLE
Subscription is not a valid Scope?

### DISCUSSION THREAD

**Discussion by catmanjan** [pnp/powershell#2542] [Q&A]
According to this page https://pnp.github.io/powershell/cmdlets/Set-PnPSearchConfiguration.html#-scope

I should be able to set search settings at a subscription level, but when I try I get this error:

Set-PnPSearchSettings : Cannot bind parameter 'Scope'. Cannot convert value "Subscription" to type
"PnP.PowerShell.Commands.Enums.SearchSettingsScope". Error: "Unable to match the identifier name Subscription to a
valid enumerator name. Specify one of the following enumerator names and try again:
Site, Web"
At line:1 char:30
+ Set-PnPSearchSettings -Scope Subscription -SearchPageUrl https://icog ...
+                              ~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Set-PnPSearchSettings], ParameterBindingException
    + FullyQualifiedErrorId : CannotConvertArgumentNoMessage,PnP.PowerShell.Commands.Search.SetSearchSettings

**Comment by AndersRask** [comment]
Probably an example that has not been updated.

If you look in the code for the enum, there is no Subscription value
https://github.com/pnp/PnP-PowerShell/blob/master/Commands/Enums/SearchSettingsScope.cs

**Reply by catmanjan** [reply]
Thanks, whats the difference between Site and Web?

Is there a way to apply my changes to all existing and new sites?
